client/src/main/com/sinch/sdk/auth/adapters/BasicAuthManager.java

```diff
@@ -47,4 +51,10 @@ public Collection<Pair<String, String>> getAuthorizationHeaders(
                 + " "
                 + Base64.getEncoder().encodeToString(raw.getBytes(StandardCharsets.UTF_8))));
   }
+
+  public boolean validateAuthenticatedRequest(
+      String method, String path, Map<String, String> headers, String jsonPayload) {
+    LOGGER.severe("checkAuthentication not implemented");
+    return false;
```

Comment:
If we receive a callback request authenticated with Basic authentication, does it mean it will be rejected b y the validation?

Reply:
Right. Currently not implemented (nor detected as used by Numbers, SMS, Verification Voice)
Could be enhanced if required.